extensions/smallrye-stork/runtime/src/main/java/io/quarkus/stork/NetworkInterfaceWrapper.java

```diff
@@ -0,0 +1,13 @@
+package io.quarkus.stork;
+
+import java.net.InetAddress;
+import java.net.SocketException;
+import java.util.Enumeration;
+
+public interface NetworkInterfaceWrapper {
```

Comment:
I assume this is part of the API, so it needs Javadoc on the class and the methods